cpp/tests/community/ecg_test.cu

```diff
@@ -54,8 +46,7 @@ TEST(ecg, success)
   cugraph::experimental::GraphCSRView<int, int, float> graph_csr(
     offsets_v.data().get(), indices_v.data().get(), weights_v.data().get(), num_verts, num_edges);
 
-  ASSERT_NO_THROW(
-    (cugraph::nvgraph::ecg<int32_t, int32_t, float>(graph_csr, .05, 16, result_v.data().get())));
+  cugraph::nvgraph::ecg<int32_t, int32_t, float>(graph_csr, .05, 16, result_v.data().get());
```

Comment:
remove nvgraph namespase, ecg should not be in nvgraph

Reply:
(this may be added to another PR in 0.15)

Reply:
Done